storage/src/metadata/storage.rs

```diff
@@ -14,13 +15,14 @@ pub struct Metadata<E: Clock + Storage + Metrics, K: Array> {
     context: E,
 
     // Data is stored in a BTreeMap to enable deterministic serialization.
-    data: BTreeMap<K, Vec<u8>>,
+    map: BTreeMap<K, Vec<u8>>,
     cursor: usize,
     partition: String,
-    blobs: [(E::Blob, u64, u64); 2],
+    blobs: [(E::Blob, u64, Vec<u8>); 2],
```

Comment:
I think it might make the code more readable if we make a struct and use it here instead of this triplet